plugins/aws_a.js

```diff
@@ -0,0 +1,16 @@
+var hoverZoomPlugins = hoverZoomPlugins || [];
+hoverZoomPlugins.push({
+    name:'AWS_a',
+    version:'0.1',
+    prepareImgLinks:function (callback) {
+        var res = [];
+
+        hoverZoom.urlReplace(res,
+            'img[src*="amazonaws"],[style*="amazonaws"]',
+            /\?.*/,
```

Comment:
@GrosPoulet This pattern seems very broad. Have you confirmed that it doesn't cause any problems with using https://aws.amazon.com console?

Reply:
It removes trailing crop/resize instructions, for instance:
https://hips.hearstapps.com/hmg-prod.s3.amazonaws.com/images/screen-shot-2021-04-05-at-12-04-54-pm-1617649515.png?crop=0.5625xw:1xh;center,top&resize=270:*
becomes:
https://hips.hearstapps.com/hmg-prod.s3.amazonaws.com/images/screen-shot-2021-04-05-at-12-04-54-pm-1617649515.png

But you're right, i'll add "**crop**" & "**resize**" keywords to make it more selective.